tests/phpunit/Mutator/ConditionalNegotiation/GreaterThanNegotiationTest.php

```diff
@@ -68,5 +68,13 @@ public static function mutationsProvider(): iterable
                 PHP
             ,
         ];
+
+        yield 'It does not mutate inside ternary to prevent overlap with TernaryMutator' => [
+            <<<'PHP'
+                <?php
+                $x > 6 ? 'yes' : 'no';
```

Comment:
here I'm not sure it is the same as e.g. #2140

To not overlap with ternary, we would need to negate `$x > 6`, isn't it?

But negation of `$x > 6` is a `$x <= 6` which isn't happening here.

So from my POV this doesn't overlap. If you don't agree, then let's find a more conrete example, probably I miss something

Reply:
I had a similar thinking at first. but I think you (like me previously) mixed up this mutator with the ones in `src/Mutator/ConditionalBoundary`

Reply:
hah, you are right. Thanks for clarifying!